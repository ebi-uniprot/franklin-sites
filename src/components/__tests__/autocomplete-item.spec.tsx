import { render } from '@testing-library/react';

import AutocompleteItem from '../autocomplete-item';

describe('AutocompleteItem component', () => {
  it('should render', () => {
    const item = {
      id: 'item id',
      pathLabel: 'A path to > some item',
      itemLabel: 'some item',
      value: 'item',
    };
    const { asFragment } = render(
      <AutocompleteItem
        item={item}
        active={false}
        substringToHighlight="item"
        key="key"
        handleOnClick={(d) => d}
      />
    );
    expect(asFragment()).toMatchSnapshot();
  });
});
