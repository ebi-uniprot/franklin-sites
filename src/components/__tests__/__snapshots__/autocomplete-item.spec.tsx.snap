// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AutocompleteItem component should render 1`] = `
<DocumentFragment>
  <li
    data-testid="autocomplete-item"
  >
    <button
      class=""
      type="button"
    >
      A path to &gt; some 
      <mark>
        item
      </mark>
    </button>
  </li>
</DocumentFragment>
`;
